packages/office-ui-fabric-react/src/components/ResizeGroup/ResizeGroup.types.ts

```diff
@@ -19,6 +19,7 @@ export interface IResizeGroupProps extends React.HTMLAttributes<ResizeGroupBase
 
   /**
    * Call to provide customized styling that will layer on top of the variant rules
+   * @deprecated
```

Comment:
Please write a more elaborate context after `@deprecated`. In this case it will no longer be respected and should be call out.

@micahgodbolt Are there scenarios you can think of where people would have used the styles prop?

Reply:
@cliffkoh not really. In practice you probably shouldn't be messing with the ResizeGroup styles. It's a utility component, not a styling one.

Reply:
Oh, to that point though, @leddie24 can you add a deprecation warning to the base component to make sure people are warned about the change in functionality? This way, if they DID use it for some reason, they can add them via className.